GNUmakefile

```diff
@@ -153,3 +154,24 @@ check-changelog-entry-file:
 .PHONY: jira-release-version
 jira-release-version:
 	go run ./tools/jira-release-version/*.go
+
+.PHONY: enable-advancedclustertpf
+enable-advancedclustertpf:
+	make delete-lines filename="./internal/provider/provider_sdk2.go" delete="mongodbatlas_advanced_cluster"
+	make add-lines filename=./internal/provider/provider.go find="project.Resource," add="advancedclustertpf.Resource,"
+	make add-lines filename=./internal/provider/provider.go find="project.DataSource," add="advancedclustertpf.DataSource,"
+	make add-lines filename=./internal/provider/provider.go find="project.PluralDataSource," add="advancedclustertpf.PluralDataSource,"
+
+.PHONY: delete-lines ${filename} ${delete}
+delete-lines:
+	rm -f file.tmp
+	grep -v "${delete}" "${filename}" > file.tmp
+	mv file.tmp ${filename}
+	goimports -w ${filename}
+
+.PHONY: add-lines ${filename} ${find} ${add}
+add-lines:
+	rm -f file.tmp
+	sed 's/${find}/${find}${add}/' "${filename}" > "file.tmp"
```

Comment:
what about -i to do it `inplace` instead?

Reply:
I tried but didn't get it, I think the problem is that the syntax is different in Mac and Linux